Refresh.GameServer/Importing/Importer.cs

```diff
@@ -59,6 +59,41 @@ private static bool MatchesMagic(ReadOnlySpan<byte> data, ulong magic)
         BitConverter.TryWriteBytes(magicSpan, BinaryPrimitives.ReverseEndianness(magic));
         return MatchesMagic(data, magicSpan);
     }
+
+    /// <summary>
+    /// Tries to detect TGA files sent from the PSP
+    /// </summary>
+    /// <param name="data"></param>
+    /// <returns></returns>
+    private static bool IsPspTga(ReadOnlySpan<byte> data)
```

Comment:
Another safety check we could add is potentially bringing in the GameToken to check it's version & platform to see if it matches with PSP.

Mainline & Vita will never upload TGA files.